spec/spec_helper.rb

```diff
@@ -9,5 +9,11 @@
   config.cassette_library_dir = File.join(ManageIQ::Providers::Nuage::Engine.root, 'spec/vcr_cassettes')
 end
 
+ALL_REFRESH_SETTINGS = [
```

Comment:
Perhaps move this into `refresher_spec`? I don't think helper is the correct place for this.

Reply:
Done. I was in doubt where to put this in first place, so I'm happy you've pointed it out.